### PR TITLE
Adding config for display of table headers in Console and removing the description display

### PR DESCRIPTION
**Pull request by malshan1998**
## Purpose
This PR adds a config for the display of table headers in the application listing component of the Console app. Furthermore, it removes the display of the description as well as the pop-up feature when the character limit is exceeded.

#### Config for display of table headers:

The display of table headers was not previously enabled in the listing API, since it had only one column. But by the PR https://github.com/wso2/identity-apps/pull/3282, another column was added to display the template names. Therefore the possibility of display of table headers was required.

Screenshot 2022-08-29 at 07 20 51

The default value is set to false, since the tables in **Manage** tab of product-is Console do not display table headers (to maintain the consistency)

#### Removal of description

The display of description in the listing UI was considered unnecessary under discussions. Therefore, the display of description as well as the pop-op feature was removed. Previously, it was designed to give a pop-up when hovering over the description, if the description had exceeded the maximum limit of 80 characters.

Before:

Screenshot 2022-08-29 at 07 21 31

After:

Screenshot 2022-08-29 at 07 20 51

### Related Issues
- Epic - https://github.com/wso2/product-is/issues/10681

### Related PRs
- Related PR `#1` - https://github.com/wso2/identity-apps/pull/3282

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/extensions/configs/application-list.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
```
Comment (jeradrutnam) [inline apps/console/src/extensions/configs/application-list.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
```

File: apps/console/src/extensions/configs/models/application-list.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
```
Comment (jeradrutnam) [inline apps/console/src/extensions/configs/models/application-list.ts:2]:
```suggestion
 * Copyright (c) 2021, WSO2 LLC. (http://www.wso2.com). All Rights Reserved.
```

**Review by brionmario — APPROVED** [review]

File: modules/i18n/src/models/namespaces/console-ns.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
Comment (brionmario) [inline modules/i18n/src/models/namespaces/console-ns.ts:2]:
```suggestion
* Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
```

File: apps/console/src/features/applications/components/application-list.tsx [hunk 4]
```diff
@@ -64,6 +64,7 @@ import {
     ApplicationTemplateListItemInterface
 } from "../models";
 import { ApplicationTemplateManagementUtils } from "../utils";
+import { applicationListConfig } from "../../../extensions/configs/application-list";
```
Comment (brionmario) [inline apps/console/src/features/applications/components/application-list.tsx:67]: Lets fix these @pavinduLakshan can you assist?

Comment (pavinduLakshan) [reply apps/console/src/features/applications/components/application-list.tsx:67]: Sure

File: modules/i18n/src/translations/si-LK/portals/console.ts [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
Comment (brionmario) [inline modules/i18n/src/translations/si-LK/portals/console.ts:2]:
```suggestion
* Copyright (c) 2020, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
```

File: modules/i18n/src/translations/fr-FR/portals/console.ts [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
Comment (brionmario) [inline modules/i18n/src/translations/fr-FR/portals/console.ts:2]:
```suggestion
 * Copyright (c) 2020, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
```

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 7]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
Comment (brionmario) [inline modules/i18n/src/translations/en-US/portals/console.ts:2]:
```suggestion
 * Copyright (c) 2020, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
```

File: modules/i18n/src/models/namespaces/console-ns.ts (hunk 3 above)
Comment (brionmario) [inline modules/i18n/src/models/namespaces/console-ns.ts:2]:
```suggestion
 * Copyright (c) 2020, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
```

File: apps/console/src/features/applications/components/application-list.tsx [hunk 8]
```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
Comment (brionmario) [inline apps/console/src/features/applications/components/application-list.tsx:2]:
```suggestion
 * Copyright (c) 2020, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
```

File: apps/console/src/extensions/configs/application-list.ts [hunk 9]
```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
```
Comment (brionmario) [inline apps/console/src/extensions/configs/application-list.ts:2]:
```suggestion
 * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
```

File: apps/console/src/extensions/configs/models/application-list.ts [hunk 10]
```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
```
Comment (brionmario) [inline apps/console/src/extensions/configs/models/application-list.ts:2]:
```suggestion
 * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
 ```

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/applications/components/application-list.tsx (hunk 4 above)
Comment (malshan1998) [reply apps/console/src/features/applications/components/application-list.tsx]: Fixed with commit fbc9b4d354e56a75b1814bd110884160c12b6358

File: modules/i18n/src/translations/fr-FR/portals/console.ts (hunk 6 above)
Comment (malshan1998) [reply modules/i18n/src/translations/fr-FR/portals/console.ts]: Fixed with commit 5c40bad7edf313a3c38f56c509339acc7b3ca010

File: modules/i18n/src/translations/si-LK/portals/console.ts (hunk 5 above)
Comment (malshan1998) [reply modules/i18n/src/translations/si-LK/portals/console.ts]: Fixed with commit 5c40bad7edf313a3c38f56c509339acc7b3ca010

File: modules/i18n/src/translations/en-US/portals/console.ts (hunk 7 above)
Comment (malshan1998) [reply modules/i18n/src/translations/en-US/portals/console.ts]: Fixed with commit 5c40bad7edf313a3c38f56c509339acc7b3ca010

File: modules/i18n/src/models/namespaces/console-ns.ts (hunk 3 above)
Comment (malshan1998) [reply modules/i18n/src/models/namespaces/console-ns.ts]: Fixed with commit 5c40bad7edf313a3c38f56c509339acc7b3ca010

File: apps/console/src/features/applications/components/application-list.tsx (hunk 8 above)
Comment (malshan1998) [reply apps/console/src/features/applications/components/application-list.tsx]: Fixed with commit 5c40bad7edf313a3c38f56c509339acc7b3ca010

File: apps/console/src/extensions/configs/models/application-list.ts (hunk 10 above)
Comment (malshan1998) [reply apps/console/src/extensions/configs/models/application-list.ts]: Fixed with commit e9254868d3563932ffb738d336609959d39f6964

File: apps/console/src/extensions/configs/application-list.ts (hunk 9 above)
Comment (malshan1998) [reply apps/console/src/extensions/configs/application-list.ts]: Fixed with commit e9254868d3563932ffb738d336609959d39f6964

**Pull request merged**